app/controllers/users_controller.rb

```diff
@@ -1,29 +1,24 @@
 class UsersController < ApplicationController
-  LOCALES = [
-              ["English", "en"],
-            ]
-
-  def settings
-    load_locales
+  def edit
```

Comment:
Add a check to ensure that the current user can actually edit this user. We might need a new `user` resource and its corresponding `user:update` action. Same applies to the `update` action in this controller.